### PR TITLE
[Anti2re] added ptrace functionality

### PR DESCRIPTION
**Pull request by VishnuPratapagiri**

### CODE REVIEW

**Comment by adityavardhanpadala** [comment]
Looks like we are replacing the function call with a dummy function. But the issue I see here with this patching method is what would we do if the function's return value saved into a variable affects control flow of the tracee. @Freakston looks like we need to have another look at what all targets are we thinking to bypass using Anti2RE

**Comment by Freakston** [comment]
The rest of the code looks fine. Do we proceed and merge it? We will have to look into this module at the end anyways. What do you guys say? @adityavardhanpadala @akulpillai

File: Anti2RE/makefile.rules [hunk 1]
```diff
@@ -0,0 +1,74 @@
+##############################################################
```
Comment (Freakston) [inline Anti2RE/makefile.rules:1]: why delete and recreate files? just asking

**Review by Freakston — CHANGES_REQUESTED** [review]
Please remove the binary

**Comment by akulpillai** [comment]
@Freakston yup, sounds good. Merge it after he removes the unwanted files

**Comment by akulpillai** [comment]
this done?

**Comment by adityavardhanpadala** [comment]
Nah it isn't. Not update on the pr after that

**Comment by akulpillai** [comment]
close this? I assume this is redundant stuff from #32 ?

**Comment by adityavardhanpadala** [comment]
okay

**Pull request closed without merging**